specification/app/resource-manager/Microsoft.App/preview/2023-05-02-preview/Builders.json

```diff
@@ -1,501 +0,0 @@
-{
```

Comment:
Why are these resource types being removed?

Reply:
Thanks for your comment. Please see my other message.